node/core/candidate-validation/src/metrics.rs

```diff
@@ -77,9 +77,12 @@ impl Metrics {
 		}
 	}
 
-	pub fn observe_pov_size(&self, pov_size: usize) {
+	pub fn observe_pov_size(&self, pov_size: usize, compressed: bool) {
 		if let Some(metrics) = &self.0 {
-			metrics.pov_size.observe(pov_size as f64);
+			metrics
+				.pov_size
+				.with_label_values(&[if compressed { "true" } else { "false" }])
```

Comment:
shouldn't the labelled values be `compressed` and `uncompressed` instead of true, false ?

Reply:
The label is `compressed` with 2 possible values: `true/false`.